packages/edit-site/src/components/site-hub/index.js

```diff
@@ -62,8 +62,7 @@ const SiteHub = memo(
 						) }
 					>
 						<Button
-							// TODO: Switch to `true` (40px size) if possible
-							__next40pxDefaultSize={ false }
+							__next40pxDefaultSize
```

Comment:
This can be seen when you visit and check for go to back button on top left corver in site editor mode.

**Note:** It has fixed height, so changing this does not change before and after images.

| Before | After     |
| :-------- | :------- |
| <img width="419" alt="site-hub-dashboard-before" src="https://github.com/user-attachments/assets/617868cd-1906-4cea-b331-bda59066c19d"> | <img width="419" alt="site-hub-dashboard-after" src="https://github.com/user-attachments/assets/3bb022cc-4c96-43bc-93e4-2345496db21c"> |